packages/spec-parser/src/specParser.ts

```diff
@@ -479,6 +480,65 @@ export class SpecParser {
     return result;
   }
 
+  async generateAdaptiveCardInPlugin(
+    pluginFilePath: string,
+    filter: string[],
+    signal?: AbortSignal
+  ): Promise<void> {
+    const newSpecs = await this.getFilteredSpecs(filter, signal);
+    const newSpec = newSpecs[1];
+    const apiPlugin = (await fs.readJSON(pluginFilePath)) as PluginManifestSchema;
+
+    const paths = newSpec.paths;
+    for (const pathUrl in paths) {
+      const pathItem = paths[pathUrl];
+      if (pathItem) {
+        const operations = pathItem;
+        for (const method in operations) {
+          if (this.options.allowMethods.includes(method)) {
+            const operationItem = (operations as any)[method] as OpenAPIV3.OperationObject;
+            if (operationItem) {
+              if (this.options.allowResponseSemantics) {
```

Comment:
we can move this condition to the top of this function

Reply:
Updated, thanks!